src/controllers/viewFinalConfirmation.controller.ts

```diff
@@ -26,7 +29,9 @@ export class ViewFinalConfirmationController extends BaseController {
 
     const viewModel: ViewModel = {
       applicationReferenceNumber: this.getApplicationReferenceNumber(),
-      officerType: dissolutionSession.officerType!
+      officerType: dissolutionSession.officerType!,
+      paymentType: dissolutionSession.paymentType!,
+      isMultiDirector: !!dissolutionSession.selectSignatoriesForm
```

Comment:
isn't there a isMultiDirector value in session? Just thinking of an applicant who logs in and pays. They will never hit the selectSignatoriesForm and this won't be present in session